service/src/test/java/bio/terra/cbas/runsets/monitoring/TestSmartRunsPoller.java

```diff
@@ -198,4 +199,58 @@ void updateNewlyCompletedRuns() throws Exception {
         COMPLETE,
         actual.stream().filter(r -> r.id().equals(completedRunId)).toList().get(0).status());
   }
+
+  @Test
+  void updatingOutputFails() throws Exception {
```

Comment:
TOL/Nit: Do you also want to check that any other workflows around the failing update still manage to succeed?

Reply:
I have updated this test to test this scenario.